scheduler/scheduler-server/src/main/java/org/ow2/proactive/scheduler/synchronization/AOSynchronization.java

```diff
@@ -128,6 +128,15 @@ public AOSynchronization() {
         // ProActive empty no arg constructor
     }
 
+    public void freeze() throws IOException {
+        recordManager.close();
```

Comment:
some work needs to be done in runActivity method in order to make freeze coherent inside the api. 
Right now, after calling freeze, all other methods will fail. I would prefer that they are not executed until resume is called.